VK_App/VK_App/Controller/FriendsViewController.swift

```diff
@@ -3,75 +3,187 @@
 
 import UIKit
 
+typealias Info = [Character: [(String, String)]]
```

Comment:
```suggestion
typealias Info = [Character: [(String, String)]]
```
Можно быть внутри контролера а не глобальным